### PR TITLE
Shipment calculator

### PR DESCRIPTION
**Pull request by ykaya932**
@goksubaser and I created an API for estimating shipment prices. During the implementation, we have used [Geolocation API](https://developers.google.com/maps/documentation/geolocation/intro)

### CODE REVIEW

**Review by afraarslan — COMMENTED** [review]
You write 'shipment_calculator' into the installed app. I saw it in [Django tutorials](https://docs.djangoproject.com/en/3.0/intro/tutorial02/) used like 'app_name.apps.AppNameConfig',. For your app, I think 
'shipment_calculator.apps.ShipmentCalculatorConfig' the way it is written. Is this another way to write it?

File: practice_app/shipment_calculator/views.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        form = VendorForm(request.POST)
+        if form.is_valid():
+            try:
+                form.save()
+                return redirect('/shipment_calculator/add_vendor/')
+            except:
+                pass
+    else:
+        form = VendorForm()
+    return render(request, 'shipment_calculator/add_vendor.html', {'form': form})
```
Comment (afraarslan) [inline practice_app/shipment_calculator/views.py:19]: I couldn't find the add_vendor.html. Did I miss it?

Comment (nursimacelik) [reply practice_app/shipment_calculator/views.py:19]: I see it here 'practice_app/shipment_calculator/templates/shipment_calculator/add_vendor.html'.

**Comment by nursimacelik** [comment]
> Is this another way to write it?

From [this answer](https://stackoverflow.com/questions/34377237/how-does-it-work-the-naming-convention-for-django-installed-apps) in stackoverflow, both namings are acceptable, foo.apps.FooConfig is being the recommended one.

- If you add in INSTALLED_APPS the typename foo.apps.FooConfig, then that class will be used to setup the foo app, 1.7 style (recommended).

- If you add in INSTALLED_APPS the plain name foo, then:
        if there is a variable foo.default_app_config this class will be used to setup the foo app, 1.7 style. Most (all?) the standard Django apps have this variable, so that you don't need to change your INSTALLED_APPS when you upgrade from Django-1.6 to Django-1.7.
        if there is not such a variable, then the 1.6 style application will be used, with default values for the advanced configuration options."

**Review by nursimacelik — APPROVED** [review]
Everyting looks fine.

**Pull request merged**